setup.py

```diff
@@ -1,57 +0,0 @@
-# This file is part of visier-connector-python.
-#
-# visier-connector-python is free software: you can redistribute it and/or modify
-# it under the terms of the Apache License, Version 2.0 (the "License").
-#
-# visier-connector-python is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
-# Apache License, Version 2.0 for more details.
-#
-# You should have received a copy of the Apache License, Version 2.0
-# along with visier-connector-python. If not, see <https://www.apache.org/licenses/LICENSE-2.0>.
-
-"""
-Visier Python Connector
-"""
-
-from setuptools import setup, find_packages  # noqa: H301
-import visier
-
-
-NAME = "visier-connector"
-VERSION = visier.__version__
-
-# To install the library, run the following
```

Comment:
This comment didn't make it in to the new file. Is this or any other comments required?

Reply:
Correct. I don't think it would add anything meaningful